axelor-human-resource/src/main/java/com/axelor/apps/hr/service/timesheet/TimesheetServiceImpl.java

```diff
@@ -943,7 +943,7 @@ public void generateLinesFromExpectedProjectPlanning(Timesheet timesheet) throws
                       + "(SELECT timesheetLine.projectPlanningTime.id FROM TimesheetLine as timesheetLine "
                       + "WHERE timesheetLine.projectPlanningTime != null "
                       + "AND timesheetLine.timesheet = ?3) "
-                      + "AND self.timeSpentTask is null",
+                      + "AND self.task is null",
```

Comment:
remove 'self.task is null', because task can be null incase when planningTime line created from project